python/tests/test_options.py

```diff
@@ -27,98 +31,165 @@ def test_accepts_browser_version_platform_name(self):
         assert sauce.platform_name == 'macOS 10.13'
 
     def test_accepts_w3c_values(self):
-        options = {
-            "browserName": "firefox",
-            "sauce:options": {
-                "browserName": "firefox",
-                "browserVersion": "73.0",
-                "platformName": "Windows 8.1"
-            }
-        }
+        sauce = SauceOptions(acceptInsecureCerts=True, pageLoadStrategy='eager')
+
+        assert sauce.accept_insecure_certs == True
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4711738)